website/src/models/group.js

```diff
@@ -168,6 +170,64 @@ schema.methods.isMember = function isGroupMember (user) {
   }
 };
 
+schema.methods.startQuest = function startQuest () {
+  if (this.type !== 'party') throw new BadRequest('Must be a party to use this method');
+  if (!this.quest.key) throw new BadRequest('Party does not have a pending quest');
+  if (this.quest.active) throw new BadRequest('Quest is already active');
```

Comment:
@crookedneighbor I think we need them on the controller probably but here yeah they don't make much sense, I'd keep them only if the startQuest would become an async function / promise so that they would be passed to the errorHandler middlewares